src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixMultipleCodeAction.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CodeFixes
+{
+    /// <summary>
+    /// Fix multiple occurrences code action.
+    /// </summary>
+    internal partial class FixMultipleCodeAction : FixAllCodeAction
+    {
+        private readonly string _title;
+        private readonly string _previewChangesDialogTitle;
+        private readonly string _computingFixWaitDialogMessage;
+
+        internal FixMultipleCodeAction(FixMultipleContext fixMultipleContext, FixAllProvider fixAllProvider, string title, string previewChangesDialogTitle, string computingFixWaitDialogMessage)
```

Comment:
I am going to retain the current type hierarchy and file a separate issue to design and cleanup all the type hierarchy in the fix system: fix contexts, suggested actions, etc. Even Cyrus was interested in cleaning this up.